app.json

```diff
@@ -3,9 +3,6 @@
   "description": "",
   "scripts": {},
   "env": {
-    "API_URL": {
```

Comment:
also this should not be removed

Reply:
same here

Reply:
wait, actually we can remove this, we dont use API_URL anymore